.dockerignore

```diff
@@ -0,0 +1,6 @@
+*
+!*.js
```

Comment:
does this ignore ALL js files?

Reply:
No, on the contrary, this command is to not ignore those files.